core/backend/src/IModelDb.ts

```diff
@@ -2007,7 +2011,8 @@ export namespace IModelDb { // eslint-disable-line no-redeclare
         throw new IModelError(ret.error.status, `TreeId=${treeId} TileId=${tileId}`);
       }
 
-      return ret.result!;
```

Comment:
this is a change to not throw and return undefined in production. I don't know what's right. @pmconne

Reply:
`ret` is an `ErrorStatusOrResult`. We failed to define that type in such a way as to convey that it always holds exactly one of an error or a result, so he's asserting it.

Reply:
my bad, i somehow thought he had removed the check for `ret.error`